### PR TITLE
feat: add dynamic signup button color

### PR DESCRIPTION
**Pull request by ayadotdev**
## Changes

This PR adds dynamic signup button color class.

Screenshot 2022-03-01 at 09 56 38

## Manual Testing

- On those affected packages:
- [x] Have you done sanity checks in the webapp?
- [x] Have you done sanity checks in the extension?

DD-407 #done

### CODE REVIEW

File: packages/shared/src/components/LoginButton.tsx [hunk 1]
```diff
@@ -43,7 +48,7 @@ export default function LoginButton({
     <Button
       onClick={onClick}
       icon={icon}
-      className={classNames('btn-primary', className)}
+      className={classNames('btn-primary', className, buttonColor.button)}
```
Comment (rebelchris) [inline packages/shared/src/components/LoginButton.tsx:51]: This buttonColor should actually overwrite the btn-primary, else we have contradicting classes.

File: packages/shared/src/lib/featureManagement.ts [hunk 2]
```diff
@@ -6,6 +6,11 @@ export class Features {
     'Access all features',
   );
 
+  static readonly SignupButtonColor = new Features(
+    'signup_button_theme_color',
+    'primary',
+  );
```
Comment (rebelchris) [inline packages/shared/src/lib/featureManagement.ts:12]: Think about this, something I forgot to mention is that we now also have `validOptions` as the third parameter here.
This should be an array of the options we allow to be valid.

```suggestion
  static readonly SignupButtonColor = new Features(
    'signup_button_theme_color',
    'primary',
    [
    'primary',
    'color1',
    'color2',
  ]
  );
```

**Review by rebelchris — CHANGES_REQUESTED** [review]
Awesome, looks great just two minor changes and we should be ready to go

**Review by rebelchris — APPROVED** [review]
Nice work!
@sshanzel Would you mind having a look before we merge

**Review by sshanzel — APPROVED** [review]
Looks perfect to my eyes! 🤩 🚢

**Pull request merged**